dcpy/builds/plan.py

```diff
@@ -26,18 +26,25 @@ class RecipeInputsVersionStrategy(str, Enum):
     copy_latest_release = "copy_latest_release"
 
 
-class DataPreprocessor(BaseModel, use_enum_values=True, extra="forbid"):
+class DataPreprocessor(BaseModel, extra="forbid"):
```

Comment:
I can't quite recall why `use_enum_values=True` was used, but it might have been that otherwise, when we deserialize the recipe into a yaml file, it these values aren't output as strings, but python classes. In any case, curious if this was causing problems?

Reply:
Not quite sure why/how I took this one out to be honest. This class doesn't have any enums so it isn't useful, but it certainly wasn't doing any harm.